src/main/java/teammates/ui/controller/AdminSessionsPageAction.java

```diff
@@ -239,50 +239,27 @@ private AccountAttributes getRegisteredInstructorAccountFromInstructors(List<Ins
     }
 
     private int getTotalNumOfOpenStatusSession(List<FeedbackSessionAttributes> allOpenFeedbackSessionsList) {
-
-        int numOfTotal = 0;
-        for (FeedbackSessionAttributes sessionAttributes : allOpenFeedbackSessionsList) {
-            if (sessionAttributes.isOpened()) {
-                numOfTotal += 1;
-            }
-        }
-
-        return numOfTotal;
+        return (int) allOpenFeedbackSessionsList.stream()
+                .filter(sessionAttributes -> sessionAttributes.isOpened())
+                .count();
```

Comment:
can change all the `total...` private variables to type `long` and avoid this awkward casting?